hooks/OystHookGetConfigurationProcessor.php

```diff
@@ -37,7 +35,10 @@ class OystHookGetConfigurationProcessor extends FroggyHookProcessor
         'FC_OYST_REDIRECT_ERROR'          => 'string',
         'FC_OYST_REDIRECT_SUCCESS_CUSTOM' => 'string',
         'FC_OYST_REDIRECT_ERROR_CUSTOM'   => 'string',
-        'FC_OYST_API_KEY'                 => 'string',
+        'FC_OYST_API_PROD_KEY' => 'string',
```

Comment:
Alright for the sort key, could improve readabilty of the code but its useless to keep alignment
- Wasting time to keep everything, every time on the same col specially when you made change
- Beside, not really a good practice at all.